src/list/whole/view.rs

```diff
@@ -843,10 +871,34 @@ where
             }
         }
     }
-}
 
-fn to_unchecked_html(display: &str) -> VNode {
-    Html::from_html_unchecked(AttrValue::from_str(display).expect("AttrValue never returns Err."))
+    /// # Panics
+    ///
+    /// `AttrValue` never returns Err
+    pub fn to_unchecked_html(
```

Comment:
This method doesn't need to be public. Since only public methods are required to include Rust documentation, the comments will be unnecessary.

Reply:
Done